index.js

```diff
@@ -39,4 +39,75 @@ searchBtnCmp.addEventListener("click", function() {
             weatherWindCmp.appendChild(newWindVal);
         })
         .catch(error => alert("Wrong City Name"))
-});
\ No newline at end of file
+});
+const mondayCmp = document.getElementById("selectMonday");
+const daysList = document.getElementById("getDaysList");
+const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thur', 'Fri', 'Sat'];
+const nextDays = 4;
+
+function createDays() {
+    let today = new Date();
+    let daysSorted = [];
+    for (let i = 0; i < nextDays; i++) {
+        let newDate = new Date(today.setDate(today.getDate() + 1));
+        daysSorted.push(days[newDate.getDay()]);
+    }
+    for (let i = 0; i < daysSorted.length; i++) {
+        let newLiElement = document.createElement("li");
+        newLiElement.textContent = daysSorted[i];
+        daysList.appendChild(newLiElement);
+    }
```

Comment:
you can do the same thing with only one for loop
```js
    for (let i = 0; i < nextDays; i++) {
        let newDate = new Date(today.setDate(today.getDate() + 1));
        let newLiElement = document.createElement("li");
        newLiElement.textContent = days[newDate.getDay()];
        daysList.appendChild(newLiElement);
    }
```
so you don't really need to populate the daysSorted first and then iterate on it.